airflow/www/static/css/graph.css

```diff
@@ -136,6 +136,10 @@ g.node.queued rect {
   stroke: grey;
 }
 
+g.node.scheduled rect{
+  stroke: lightgreen;
```

Comment:
I believe that the scheduled color is `tan`. We already use `lime` for running `lightgreen` is too similar.